pkg/services/alerting/notifiers/slack.go

```diff
@@ -110,37 +137,61 @@ func NewSlackNotifier(model *models.AlertNotification) (alerting.Notifier, error
 	username := model.Settings.Get("username").MustString()
 	iconEmoji := model.Settings.Get("icon_emoji").MustString()
 	iconURL := model.Settings.Get("icon_url").MustString()
-	mention := model.Settings.Get("mention").MustString()
+	mentionUsersStr := model.Settings.Get("mentionUsers").MustString()
+	mentionGroupsStr := model.Settings.Get("mentionGroups").MustString()
+	mentionChannel := model.Settings.Get("mentionChannel").MustString()
 	token := model.Settings.Get("token").MustString()
 	uploadImage := model.Settings.Get("uploadImage").MustBool(true)
 
+	if mentionChannel != "" && mentionChannel != "here" && mentionChannel != "channel" {
+		return nil, fmt.Errorf(fmt.Sprintf("invalid value for mentionChannel: %q", mentionChannel))
+	}
+	mentionUsers := []string{}
+	for _, u := range strings.Split(mentionUsersStr, ",") {
+		u = strings.TrimSpace(u)
+		if u != "" {
+			mentionUsers = append(mentionUsers, u)
+		}
+	}
+	mentionGroups := []string{}
+	for _, g := range strings.Split(mentionGroupsStr, ",") {
+		g = strings.TrimSpace(g)
+		if g != "" {
+			mentionGroups = append(mentionGroups, g)
+		}
+	}
+
 	return &SlackNotifier{
-		NotifierBase: NewNotifierBase(model),
-		URL:          url,
-		Recipient:    recipient,
-		Username:     username,
-		IconEmoji:    iconEmoji,
-		IconURL:      iconURL,
-		Mention:      mention,
-		Token:        token,
-		Upload:       uploadImage,
-		log:          log.New("alerting.notifier.slack"),
+		NotifierBase:   NewNotifierBase(model),
+		URL:            url,
+		Recipient:      recipient,
+		Username:       username,
+		IconEmoji:      iconEmoji,
+		IconURL:        iconURL,
+		MentionUsers:   mentionUsers,
+		MentionGroups:  mentionGroups,
+		MentionChannel: mentionChannel,
+		Token:          token,
+		Upload:         uploadImage,
+		log:            log.New("alerting.notifier.slack"),
 	}, nil
 }
 
 // SlackNotifier is responsible for sending
 // alert notification to Slack.
 type SlackNotifier struct {
 	NotifierBase
-	URL       string
-	Recipient string
-	Username  string
-	IconEmoji string
-	IconURL   string
-	Mention   string
```

Comment:
Think we need to document this as a breaking change as well.

Reply:
@marefr Is there anything I need to do in this regard?

Reply:
The breaking change is in Slacks API but our users is still affected so I think it might be a good idea to add a note about this in the changelog.

Reply:
@bergquist OK, adding an item to the changelog.

Reply:
@marefr @bergquist Added an item to 6.7.0 breaking changes in changelog. Please let me know if I should do it differently.